mlrun/__main__.py

```diff
@@ -1091,6 +1102,17 @@ def project(
                     "token": proj.get_param("GIT_TOKEN"),
                 },
             )
+        if notifications:
+            for notification in notifications:
+                if "=" in notification:
+                    file_path = notification.split("=")[-1]
```

Comment:
more explicit to check if specifically a file was requested
```suggestion
                if notification.startswith("file="):
                    file_path = notification.split("=")[-1]
```

Reply:
Fixed